### ISSUE TITLE
Single-pushing the CREDIT button gives a lot of credits

### ISSUE THREAD

**Issue by vanfanel** [twinaphex/cannonball#18]
Hi there,

Am I the only who noticed that pressing the credits button once gives live 4-9 credits?

**Comment by Tenome** [comment]
No, it just dumps me 9 credits. It's either this libretro core, or the revision they based it off of. Here's a desktop version that's much better until this core gets updated.

https://retroreloader.itch.io/outrun86coast

**Comment by Tatsuya79** [comment]
Should be fixed with https://github.com/libretro/cannonball/pull/26
